game-core/src/main/java/games/strategy/triplea/delegate/battle/MustFightBattle.java

```diff
@@ -1576,15 +1570,15 @@ public void execute(final ExecutionStack stack, final IDelegateBridge bridge) {
       }
     };
 
-    steps.add(
-        new IExecutable() {
-          private static final long serialVersionUID = 1560702114917865290L;
+    steps.add(defensiveStandard);
```

Comment:
nit, can we use a different word than `standard`? I don't know it's well enough defined to have a lot of meaning. `DefensiveRoll` would perhaps be just as meaningful. `standard` raises the question, what is non-standard?

Reply:
All of the other steps refer to their "type": FirstStrike, AA, NavalBombardment, etc.  I agree that `standard` is probably not the best, but I don't think `roll` is even better.  I could do `units` but then it might be confusing since this isn't all of the defensive units, only the ones that haven't fired yet.  I've seen the code also refer to this group as `regular`.  I'm not sure what to call them.

Reply:
Perhaps simply then: 'defensive' without the qualifier?

Reply:
The class names would be just "Defensive"?  That feels a little weird to me.

I went through a few rule books and I noticed that "general" was used a few times.  So maybe defensiveGeneral/offensiveGeneral?  Or "units" since the rule books would say "attacking units", even though it had mentioned subs, etc beforehand.

Reply:
Interesting thoughts, as another suggestion:  `DefensiveUnitsReturnFire`?

Reply:
I've renamed them to be `DefensiveGeneral` and `OffensiveGeneral` and they are in the new `general` package.

Reply:
I think I've a slight preference for the names "OffensiveFire" and "DefensiveFire". The qualified variants of that I think distinguish those, somewhat concerned that 'general' has the same issues as standard.

NBD though.